src/index.js

```diff
@@ -0,0 +1,112 @@
+import './style.css';
+import checkingList from './module/checkbox.js';
+
+let arr = [];
+
+const arrCollection = () => {
+  const names = document.querySelector('#items').value;
+  const completed = false;
+  const id = arr.length;
+  arr.push({ names, completed, id });
+};
+const arrowImage = document.querySelector('#materialistic');
+arrowImage.addEventListener('click', () => {
+  arrCollection();
+  localStorage.setItem('arr', JSON.stringify(arr));
+  window.location.reload();
+});
+arr = JSON.parse(localStorage.getItem('arr')) || [];
+
+const addRemove = () => {
```

Comment:
You can separate this function into two functions: one for add and one for remove.